QuizApp/QuizApp/Coordinator.swift

```diff
@@ -13,10 +13,18 @@ class Coordinator: CoordinatorProtocol {
         navigationController.setViewControllers([Container.loginViewController()], animated: true)
     }
 
-    func showTabBarController() {
+    func showTabBar() {
         navigationController.setViewControllers([setUpTabBarController()], animated: true)
     }
 
+    func dismiss() {
```

Comment:
Dismiss is a UIKit navigation action and you should try to hide implementation details. Naming this `hideLeaderboard` would still be correct and it would hide the details.